File: Lifecycle-aware/Exoplayer2/app/src/main/java/jalilurrahman/com/exoplayer2/MainActivity.java
package jalilurrahman.com.exoplayer2;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

import com.google.android.exoplayer2.SimpleExoPlayer;
import com.google.android.exoplayer2.ui.PlayerView;
import com.google.android.exoplayer2.ui.SimpleExoPlayerView;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        PlayerView playerView = findViewById(R.id.player_view);
        String videoUrl = "http://docs.evostream.com/sample_content/assets/bunny.mp4";

        getLifecycle().addObserver(new VideoPlayerComponent(this,playerView,videoUrl));
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: ui testing/kakaoespressosample/app/src/main/java/jalilurrahman/com/kakao_espresso_sample/MainActivity.kt
package jalilurrahman.com.kakao_espresso_sample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button_update.setOnClickListener {
            text_view_hello.text = "Hello Kakao World!"
            image_view_kakao.visibility = View.VISIBLE
        }
    }
}
